src/pages/SimpleViewer.jsx

```diff
@@ -28,6 +29,48 @@ export default function SimpleViewer(props) {
   const [showSnackbar, toggleSnackbar] = useState(false);
   const messageStore = useMessageStore();
 
+  async function askQuestion() {
+    const defaultQuestion = {
+      parent: '',
+      visibility: 1,
+      metadata: { name: `Simple Viewer Question - ${formatDateTimeNicely(new Date().toLocaleString())}` },
```

Comment:
Not sure we need the date in the name. The date is in the column right next to the name in QuestionList so its just duplicate information and makes the table more messy.

Reply:
Good point.